os/tools/set_bininfo.py

```diff
@@ -21,7 +21,9 @@
 import sys
 import string
 
-EXT_NAME = sys.argv[1]
+SOURCE_EXT_NAME = sys.argv[1]
+TARGE_EXT_NAME = "trpk"
```

Comment:
How about changing the name as `TARGET_EXT_NAME`?

Reply:
@sunghan-chang Plz, explain what trpk stands for in commit description as well as in this file.
I also think TARGET is correct.

Reply:
@jeongchanKim @davidfather Done